src/helpers/gcov.ts

```diff
@@ -0,0 +1,23 @@
+import glob from 'fast-glob'
+import { isProgramInstalled, runExternalProgram } from "./util"
+
+export function generateGcovCoverageFiles(projectRoot: string, include: string[] = [], ignore: string[] = [], gcovArgs = '',) {
+    if (!isProgramInstalled('gcov')) {
+        throw new Error('gcov is not installed, cannot process files')
+    }
+
+    const globstar = (pattern: string) => `**/${pattern}`
+    const gcovInclude = ['*.gcno', ...include].map(globstar)
+    const gcovIgnore = ['bower_components', 'node_modules', 'vendor', ...ignore].map(globstar)
+    const files = glob
+        .sync(gcovInclude, {
```

Comment:
```suggestion
    const files = await glob(gcovInclude, {
```

Asynchrony should be used when possible.[^1]

Other instances of `glob` throughout the project should conform to this change accordingly.

[^1]: https://nodejs.org/en/knowledge/getting-started/control-flow/how-to-write-asynchronous-code/